### ISSUE TITLE
`formula` prop available in Python

### ISSUE THREAD

**Issue by LiamConnors** [plotly/dash-vtk#58]
#### Description
`formula` prop available in Python, though JS formulae are unusable from Python.

#### Steps/Code to Reproduce
In the python REPL run: 
```import dash_vtk
help (dash_vtk.Calculator)
````
We see there is a prop `formula` available, but JS formulae are unusable from Python

#### Expected Results
Only have props that can be used in Python

#### Actual Results
`formula` is available.

#### Versions

Dash 2.0.0
<ipython-input-3-ecfce0f39717>:3: UserWarning: 
The dash_html_components package is deprecated. Please replace
`import dash_html_components as html` with `from dash import html`
  import dash_html_components; print("Dash Core Components", dash_html_components.__version__)
Dash Core Components 2.0.0
<ipython-input-3-ecfce0f39717>:4: UserWarning: 
The dash_core_components package is deprecated. Please replace
`import dash_core_components as dcc` with `from dash import dcc`
  import dash_core_components; print("Dash HTML Components", dash_core_components.__version__)
Dash HTML Components 2.0.0
Dash VTK 0.0.9

@alexcjohnson

**Comment by jourdain** [comment]
Good point, that class/component should be removed from dash-vtk. I guess it was ported as-is from react-vtk-js.
